scripts/steward_review.py

```
@@ -0,0 +1,313 @@
+            skipped_checks.append(
+                {"id": configured["id"], "reason": "disabled by configuration"}
+            )
+            continue
+        if execution == "sandbox" and not (
+            review["execute_contributor_code"] and review["sandbox_available"]
+        ):
+            skipped_checks.append(
+                {"id": configured["id"], "reason": "sandbox execution unavailable"}
+            )
+            continue
+        result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)